src/main/java/atlas/Atlas.java

```diff
@@ -0,0 +1,56 @@
+package atlas;
+
+import atlas.command.Command;
+import atlas.exception.AtlasException;
+import java.time.format.DateTimeParseException;
+
+/**
+ * The main class that drives the Atlas application.
+ */
+public class Atlas {
+    private final String DATA_PATH = "./data/Atlas.txt";
```

Comment:
I like how your constant name is capitalized and there was the use of underscore to separate words!